packit/constants.py

```diff
@@ -47,6 +47,31 @@
         alternative_hostname=None,
         namespace="redhat/centos-stream/rpms",
     ),
+    "cbs-automotive-sig": DistGitInstance(
```

Comment:
One cannot, unfortunately...the bin name is `cbs`

Reply:
I see, this makes it a bit more problematic then :/ adjusting the way the `DistGitInstance` constants are set up would be easy (we can reference it like you suggested, but will need to add a way to use correct binary; also ensure it is installed, but I guess that is fine, since I've seen the change among playbooks), but it totally breaks the semantic meaning of `pkg_tool` option in the config :|